training/trillium/MAXTEXT_README.md

```diff
@@ -0,0 +1,30 @@
+# Prep for Maxtext GPT3-175B workloads on GKE
+1. Clone [Maxtext](https://github.com/google/maxtext) repo and move to its directory
+```
+git clone https://github.com/google/maxtext.git
+cd maxtext
```

Comment:
Hey @suexu1025 can we mention here that we need to clone maxtext at a particular commit i.e. [maxtext](https://github.com/AI-Hypercomputer/maxtext.git)@2e1ebad7c660e45d2f020ef025d74cc90e2f0eb3. 

Given that we want to create the maxtext image from that hash.

Reply:
Added